modules/alloy/src/main/ts/ephox/alloy/api/events/NativeEvents.ts

```diff
@@ -2,36 +2,38 @@ import { Fun } from '@ephox/katamari';
 
 const constant = Fun.constant;
 
-const contextmenu = constant('contextmenu');
-const touchstart = constant('touchstart');
-const touchmove = constant('touchmove');
-const touchend = constant('touchend');
-const touchcancel = constant('touchcancel');
-const gesturestart = constant('gesturestart');
-const mousedown = constant('mousedown');
-const mousemove = constant('mousemove');
-const mouseout = constant('mouseout');
-const mouseup = constant('mouseup');
-const mouseover = constant('mouseover');
+const contextmenu = constant('contextmenu' as const);
```

Comment:
I made this change so that the value returned can be used to infer other things (e.g. the event type when used in a callback).